packages/store/addon/-private/identifiers/is-stable-identifier.ts

```diff
@@ -0,0 +1,7 @@
+import { StableRecordIdentifier, IS_IDENTIFIER } from '../ts-interfaces/identifier';
+
+export default function isStableIdentifier(
+  identifier: StableRecordIdentifier | any
```

Comment:
anything ` | any` just becomes an `any`, so we should just type it as such
![Screen Shot 2019-07-17 at 1 00 15 PM](https://user-images.githubusercontent.com/558005/61407546-d7292400-a892-11e9-95c3-f22c682df584.png)